examples/auctions/blind_auction.vy

```diff
@@ -0,0 +1,63 @@
+# Blind Auction
+# Adapted to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
+
+# Custom struct for bids
+bid: {
+    blindedBid: bytes32,
+    deposit: wei_value
+}
```

Comment:
This creates a global named `bid` that is a struct of the layout provided. We don't have support for named structs *yet*, but we're working on it! (#300 )